Makefile

```diff
@@ -12,7 +12,7 @@ IMAGE_TAG?=docker tag
 APP_NAME=elasticsearch-operator
 APP_REPO=github.com/openshift/$(APP_NAME)
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
-IMAGE_TAG=quay.io/openshift/$(APP_NAME)
+IMAGE_TAG=openshift/$(APP_NAME)
```

Comment:
I think from a build image perspective, this is only significant to local building as CI and release dont make use of the tags here. @josefkarasek this should be fine.